stackit/internal/services/argus/instance/resource_test.go

```diff
@@ -46,13 +55,23 @@ func TestMapFields(t *testing.T) {
 				PlanName:   utils.Ptr("plan1"),
 				PlanId:     utils.Ptr("planId"),
 				Parameters: &map[string]string{"key": "value"},
+				Instance: &argus.InstanceSensitiveData{
+					MetricsRetentionTimeRaw: utils.Ptr(int64(30)),
+					MetricsRetentionTime1h:  utils.Ptr(int64(15)),
+					MetricsRetentionTime5m:  utils.Ptr(int64(10)),
+				},
 			},
 			&argus.ListACLResponse{
 				Acl: &[]string{
 					"1.1.1.1/32",
 				},
 				Message: utils.Ptr("message"),
 			},
+			&argus.GetMetricsStorageRetentionResponse{
+				MetricsRetentionTimeRaw: utils.Ptr("60d"),
+				MetricsRetentionTime1h:  utils.Ptr("30d"),
+				MetricsRetentionTime5m:  utils.Ptr("7d"),
+			},
```

Comment:
These being diferent is to simulate an update, right?

Reply:
AH but there is already another test case for the update, is that one different in any way?

Reply:
Yes, that's the goal. But its a nice catch because I created a specific update test and forgot to change this back